R/constants.R

```diff
@@ -20,4 +20,14 @@ day_to_sec <- 86400
 
 missing_watersheds <- c(3589, 3850, 3678, 3582, 3615, 4226, 4227, 2336, 3448, 2162, 3474) # for runoff
 
-avg_wateruse_ltr <- 333
+avg_wateruse_ltr <- 200
+
+MGD_to_m3sec <- 0.0438
+
+BCMyr_to_m3sec <- 31.71
+
+BCMmonth_to_m3sec <- 373.36
+
+ltrday_to_m3sec <- 1.15741e-8
+
+onekmsq_to_60msq <- 2.78
```

Comment:
can you double check this?... 60m by 60m = 3600m2... then 1km2 is 1000 * 1000 = 1,000,000 m2.

1,000,000 / 3600 = 277.78    (maybe I've misunderstood the resolution of the raster?)

Reply:
Nope you are correct that is my bad, I was just off by 2 decimal places. Doe everything else look correct?